scripts/container-only/.wkdev-init

```diff
@@ -204,6 +204,20 @@ try_setup_permissions_rust_directory() {
     chown --recursive "${container_user_name}:${container_group_name}" "${rust_directory}" &>/dev/null
 }
 
+# Works around python issues when running scripts like this:
+# python3 Tools/Scripts/dump-class-layout -t WebKitBuild/JSCOnly/Debug/lib/libJavaScriptCore.so JavaScriptCore JSC::JSCell
+try_fix_lldb_py() {
+
+    local user_home
+
+    user_home=$(getent passwd "${container_user_name}" | cut -d: -f6)
+
+    sudo apt install lsb-release wget software-properties-common gnupg -y
+    wget -qO - https://apt.llvm.org/llvm.sh | sudo bash -s 19 -y
```

Comment:
1. Installing core packages doesn't belong in `.wkdev-init`, it would go in the dockerfile.
2. Downloading and installing the entire llvm toolchain is a pretty massive change... Are you saying Ubuntu's clang 18 is impossible to get working?